app/forms/trainee_form.rb

```diff
@@ -69,11 +69,7 @@ def clear_stash
 private
 
   def course_date_attribute_name_prefix
```

Comment:
do we still need this at all?

Reply:
Yeah, I think it's a fundamental part of how the code works without doing a big refactor